zos_subsystems/cics/cmci/deploy_program/deploy_program.yml

```diff
@@ -47,4 +43,4 @@
         type: CICSProgram
         resources:
           filter:
-            program: '{{ program }}'
```

Comment:
Do we want this set to be double quotes as well for consistency? Lines 36-46

Reply:
I just did enough to make `ansible-lint` happy, and that's not one of its rules.